helpers/deploymentConfig.ts

```diff
@@ -900,6 +916,38 @@ export const assets: Assets = {
       oracle: "redstone",
     },
   ],
+  zksyncsepolia: [
+    {
+      token: "WBTC",
+      address: "0xeF891B3FA37FfD83Ce8cC7b682E4CADBD8fFc6F0",
+      oracle: "chainlink",
+    },
+    {
+      token: "WETH",
+      address: "0x53F7e72C7ac55b44c7cd73cC13D4EF4b121678e6",
+      oracle: "chainlink",
+    },
+    {
+      token: "USDC",
```

Comment:
Maybe we should use `USDC.e` instead, to replicate the symbol of the underlying token on mainnet

Reply:
[b49cb79](https://github.com/VenusProtocol/oracle/pull/209/commits/b49cb79556d1b1d753ca71ffcee884ece5051d4c)